android/res/layout/fragment_editor.xml

```diff
@@ -144,9 +144,39 @@
         <include
           android:id="@+id/block_opening_hours"
           layout="@layout/item_opening_hours"/>
-        <include
-          android:id="@+id/block_phone"
-          layout="@layout/item_editor_input"/>
+        <RelativeLayout
+            android:id="@+id/block_phone"
+            style="@style/MwmWidget.Editor.MetadataBlock.Clickable">
+          <ImageView
+              style="@style/MwmWidget.Editor.MetadataIcon"
+              android:src="@drawable/ic_phone"
+              tools:ignore="ContentDescription"/>
+          <TextView
+              android:id="@+id/phone"
+              style="@style/MwmWidget.Editor.FieldLayout.EditText"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:layout_centerVertical="true"
+              android:layout_marginStart="@dimen/editor_margin_left"
+              android:layout_toLeftOf="@+id/edit_phone"
```

Comment:
Redundant "android:layout_toLeftOf".  Min api level is 21.

Reply:
@Endem10n I see that attribute `android:layout_toLeftOf` is used in fragment_editor.xml on Wi-Fi checkbox row.
Should I get replace RelativeLayout with LinearLayout to get rid of `android:layout_toLeftOf` property?
Or how to correctly layout phone icon, phone number text view, and edit phone buton in a single row?

Reply:
My point is that you already have in the next line "android:layout_toStartOf="@+id/edit_phone",  so all will work fine even without "android:layout_toLeftOf".